dbaas/dbaas-aggregator/src/main/java/com/netcracker/cloud/dbaas/exceptions/InvalidClassifierException.java

```diff
@@ -10,4 +10,11 @@ public class InvalidClassifierException extends ValidationException {
     public InvalidClassifierException(String detail, Map<String, Object> classifier, Source source) {
         super(ErrorCodes.CORE_DBAAS_4010, ErrorCodes.CORE_DBAAS_4010.getDetail(detail, classifier), source);
     }
+
+    public static InvalidClassifierException withDefaultMsg(Map<String, Object> classifier) {
+        return new InvalidClassifierException("Classifier doesn't contain all mandatory fields. " +
```

Comment:
The error message string concatenation is inefficient and hard to read; use a single string literal instead.

```suggestion
        return new InvalidClassifierException("Classifier doesn't contain all mandatory fields. If authenticating with token, namespace in classifier and in token must be equal or be in the same composite structure. Check that classifier has `microserviceName`, `scope`. If `scope` = `tenant`, classifier must contain `tenantId` property",
```

#deepseek-review:inline